### ISSUE TITLE
stuck!

### ISSUE THREAD

**Issue by SeekPoint** [songgc/loan-default-prediction#1]
rzai@rzai00:~/prj/loan-default-prediction$ python train_predict.py .
training set is ./train_v2.csv
test set is ./test_v2.csv
default classifier training...
/home/rzai/prj/loan-default-prediction/models.py:54: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy
  data['hasnull'][pd.isnull(data).any(axis=1)] = 1
